src/discord/handlers/MessageHandler.js

```diff
@@ -18,12 +19,23 @@ class MessageHandler {
       return
     }
 
+    const isReply = await this.checkReply(message)
```

Comment:
The variable should be renamed to better represent what value it actually holds, using `isKeyword` would suggest it's a boolean, however, it's actually either a `string` or `null`.